common/app/model/CardStylePicker.scala

```diff
@@ -11,22 +11,33 @@ object CardStylePicker {
 
   def apply(content: CapiContent): CardStyle = {
     val tags = content.tags.map(_.id).toSeq
-    extractCampaigns(tags) match {
-      case Nil                                                                         => CardStyle(content, TrailMetaData.empty)
-      case campaign :: Nil if campaign.id.toString.toLowerCase() == "specialreportalt" => SpecialReportAlt
-      case _                                                                           => SpecialReport
+    val campaigns = extractCampaigns(tags)
```

Comment:
The CampaignAgent fetches the campaigns from the [targeting-tool](https://targeting.gutools.co.uk/) and [extractCampaigns filters the campaigns of type Report](https://github.com/guardian/frontend/blob/bc5d6427ef3e9a4f1c0026fac240689659c3ffa2/common/app/model/CardStylePicker.scala#L30-L33) (i.e. the Special Reports and the SpecialReportAlt)